README.md

```diff
@@ -104,70 +96,13 @@ ansible-playbook -e rm_dest=~/github/rm_example \
 │                   ├── __init__.py
 │                   └── utils.py
 ├── README.md
-├── roles
-
- 
-```
-**Role directory layout**
+├── tests
 
-- `network_os`: myos
-- `resource`: interfaces
-
-```
-ansible-playbook -e rm_dest=~/github/rm_example/roles/my_role \
-                 -e structure=role \
-                 -e model=models/myos/interfaces/myos_interfaces.yml \
-                 site.yml
-```
-
-```
-roles
-└── my_role
-    ├── library
-    │   ├── __init__.py
-    │   ├── myos_facts.py
-    │   └── myos_interfaces.py
-    ├── LICENSE.txt
-    ├── module_utils
-    │   ├── __init__.py
-    │   └── network
-    │       ├── __init__.py
-    │       └── myos
-    │           ├── argspec
-    │           │   ├── facts
-    │           │   │   ├── facts.py
-    │           │   │   └── __init__.py
-    │           │   ├── __init__.py
-    │           │   └── interfaces
-    │           │       ├── __init__.py
-    │           │       └── interfaces.py
-    │           ├── config
-    │           │   ├── base.py
-    │           │   ├── __init__.py
-    │           │   └── interfaces
-    │           │       ├── __init__.py
-    │           │       └── interfaces.py
-    │           ├── facts
-    │           │   ├── base.py
-    │           │   ├── facts.py
-    │           │   ├── __init__.py
-    │           │   └── interfaces
-    │           │       ├── __init__.py
-    │           │       └── interfaces.py
-    │           ├── __init__.py
-    │           └── utils
-    │               ├── __init__.py
-    │               └── utils.py
-    └── README.md
-```
 
```

Comment:
Add ``` here to end the code snippet above. Otherwise the header below ends up part of that code snippet (click view file here in github to see what I'm talking about).

Reply:
Ack.